### PR TITLE
Python:Adding testcase for API support for capture

### PR DESCRIPTION
**Pull request by SH118415**
Co-authored-by: Darshan Vandra <dvandra.ai@gmail.com>

### CODE REVIEW

File: contrib/python/api/tests/tests.py [hunk 1]
```diff
@@ -146,6 +146,27 @@ def create_node(protocol):
...
+                                insecure=True)
+    
+        capture1 = restclient.capture_create (
+            "G.V().Has('Name', 'test', 'Type', 'netns')")
+
+        captures = restclient.capture_list()
+        self.assertGreaterEqual(len(captures), 1 , "no capture found")
+
+        for capture in captures:
+            if (capture.uuid ==capture1.uuid):
```
Comment (masco-enovance) [inline contrib/python/api/tests/tests.py]: nit: space between == and capture1.uuid

**Comment by masco-enovance** [comment]
run skydive-python-tests

File: contrib/python/api/tests/tests.py (hunk 1 above)
Comment (SH118415) [reply contrib/python/api/tests/tests.py]: Okay

**Comment by SH118415** [comment]
run skydive-python-tests

**Comment by safchain** [comment]
rerun skydive-k8s-tests

**Comment by SH118415** [comment]
run skydive-python-tests

**Comment by SH118415** [comment]
run skydive-python-tests

**Pull request merged**